Integrations/integration-C2sec.yml

```
@@ -0,0 +1,236 @@
+           name = 'C2SEC Network Services Exposed'
+       for i in range(len(keys)):
+          if isinstance(c2sec[keys[i]], list):
+            labels.append({'type': keys[i], 'value': json.dumps(c2sec)})
+          else:
+              labels.append({'type': keys[i], 'value': str(c2sec[keys[i]])})
+       return {
+            'name': name,
+            'labels': labels,
+            'rawJSON': json.dumps(c2sec),
+            'occurred': occured
+        }
+
+
+    def return_error(data):
+       """
+       Return error as result and exit
+       """
+       demisto.results({
+                'Type': entryTypes['error'],
+                'ContentsFormat': formats['text'],
+                'Contents' : data
+       })
+
+
+    # The command demisto.command() holds the command sent from the user.
+    if demisto.command() == 'test-module':
+        # This is the call made when pressing the integration test button.
+        test = addcompany('six-group.com', 'false')
+        if not isinstance(test, tuple):
+          demisto.results("ok")
+        else:
+            demisto.results(test)
+        sys.exit(0)
+    elif demisto.command() == 'fetch-incidents':
+        demisto.results(fetch_incidents())
+    elif demisto.command() == 'irisk-add-company':
+      result = addcompany(demisto.args()['domain'],demisto.args()['newscan'])
+      demisto.results(result)
+      sys.exit(0)
+    elif demisto.command() == 'irisk-query-workitem':
+      result = queryWorkitem(demisto.args()['workitemid'])
+      demisto.results(result)
+      sys.exit(0)
+    elif demisto.command() == 'irisk-rescan-company':
+      result = rescan(demisto.args()['domain'])
+      demisto.results(result)
+      sys.exit(0)
+    elif demisto.command() == 'irisk-query-issues':
+      result = queryIssues(demisto.args()['domain'])
+      demisto.results(result)
+      sys.exit(0)
+    elif demisto.command() == 'irisk-query-component':
+      result = queryComponent(demisto.args()['domain'],demisto.args()['component'])
+      demisto.results(result)
+      sys.exit(0)
+  type: python
+  commands:
+  - name: irisk-add-company
+    arguments:
+    - name: domain
+      required: true
+      description: Domain to be added into the portfolio
+    - name: newscan
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - "true"
+      - "false"
+      description: Flag to indicate whether we would always start a new scan for the
+        domain
+    description: Add a company into portfolio
+  - name: irisk-query-workitem
+    arguments:
+    - name: workitemid
+      required: true
+      description: Domain whose status to be checked
+    outputs:
+    - contextPath: c2secScanQueryStatus
+      description: Scan Query Status
+    description: Query workitem status
+  - name: irisk-rescan-company
+    arguments:
+    - name: domain
+      required: true
+      description: Domain to be rescaned
+    outputs:
+    - contextPath: c2secScanRequest
```

Are we aware of an inner field that would be relevant here?
```suggestion
    - contextPath: C2sec.ScanRequest
```

---

fixed